x/contractmanager/client/cli/query_failure.go

```diff
@@ -23,47 +24,17 @@ func CmdAllFailures() *cobra.Command {
 
 			queryClient := types.NewQueryClient(clientCtx)
 
-			params := &types.QueryAllFailureRequest{
-				Pagination: pageReq,
+			address := ""
+			if len(args) > 0 {
```

Comment:
btw is there a use of command execution with an empty address, i.e. of getting the whole failures happened in the chain? maybe the simple by-address-one is enough?

Reply:
At least you can. Do not see any issue with it.